### PR TITLE
feat(plugins): community plugin oas-fastify

### PR DESCRIPTION
**Pull request by ahmadnassri**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by RafaelGSS — APPROVED** [review]
LGTM.

File: docs/Ecosystem.md [hunk 1]
```diff
@@ -191,5 +191,6 @@ Plugins maintained by the Fastify team are listed under [Core](#core) while plug
 - [`i18next-http-middleware`](https://github.com/i18next/i18next-http-middleware#fastify-usage) An [i18next](https://www.i18next.com) based i18n (internationalization) middleware to be used with Node.js web frameworks like express or `Fastify` and also for Deno.
 - [`k-fastify-gateway`](https://github.com/jkyberneees/fastify-gateway) API Gateway plugin for `fastify`, a low footprint implementation that uses the `fastify-reply-from` HTTP proxy library.
 - [`mercurius`](https://mercurius.dev/) A fully-featured and performant GraphQL server implementation for Fastify.
+- ['oas-fastify'](https://github.com/ahmadnassri/node-oas-fastify) OAS 3.x to Fastify routes automation. Automatically generates route handlers with fastify configuration and validation.
```
Comment (jsumners) [inline docs/Ecosystem.md:194]: I believe this is out of order. It should come after `nstats` instead of before it.

Comment (ahmadnassri) [reply docs/Ecosystem.md:194]: oops, I'll correct it

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by ahmadnassri** [comment]
thanks all! enjoy your weekend!

**Pull request merged**